### PR TITLE
Symphony 2.5 compatibility

### PR DESCRIPTION
**Pull request by bernardodiasc**
This fix a issue:

`Symphony Fatal Error: Call to a member function get() on a non-object`

https://github.com/symphonycms/symphony-2/wiki/Migration-Guide-to-2.5-for-Developers#properties

Cheers!

### CODE REVIEW

File: content/content.index.php [hunk 1]
```diff
@@ -14,10 +14,17 @@ public function __viewIndex() {
...
+			$author = Symphony::Author();
+		} else {
+			$author = Administration::instance()->Author;
+		}
+
 		// Add welcome message
 		$hour = date('H');
 		$welcome = __('Nice to meet you');
-		if(Administration::instance()->Author->get('last_seen') != NULL) {
+		if($author instanceof Author && $author->get('last_seen') != NULL) {
```
Comment (nitriques) [inline content/content.index.php]: I think that `$author instanceof Author` does not have any purpose. I would also declare the $author variable properyly ($author = null, before the if)

File: content/content.index.php [hunk 2]
```diff
@@ -48,8 +55,8 @@ public function __viewIndex() {
 		
 		$actions = array();
 		$actions[] = Widget::Select('panel-type', $panel_types_options);
-		
-		if(Administration::instance()->Author->isDeveloper()) {
+
+		if(($author instanceof Author && $author->isDeveloper()) ? true : false) {
 			$actions[] = Widget::Anchor(
```
Comment (nitriques) [inline content/content.index.php]: Same thing here, `$author instanceof Author` does not have any purpose

**Comment by nitriques** [comment]
I'll be glad to pull it in :+1:

**Comment by bernardodiasc** [comment]
nice, @nitriques please check if it is good to go now!

File: content/content.index.php [hunk 3]
```diff
@@ -48,8 +56,8 @@ public function __viewIndex() {
 		
 		$actions = array();
 		$actions[] = Widget::Select('panel-type', $panel_types_options);
-		
-		if(Administration::instance()->Author->isDeveloper()) {
+
+		if($author->isDeveloper() ? true : false) {
 			$actions[] = Widget::Anchor(
```
Comment (nitriques) [inline content/content.index.php]: true : false is redundant here: isDeveloper already returns a bool

**Comment by nitriques** [comment]
Still have a little weird thing... After this, I'll pull it in! :smile: Thanks!

File: content/content.index.php (hunk 3 above)
Comment (bernardodiasc) [inline content/content.index.php]: oops, sorry ;P

**Comment by bernardodiasc** [comment]
Rebased commits, please confirm if it is ok.

FYI I'm having another issue while including blocks in the dashboard, but not sure yet if it's a compatibility issue. I'll look further into this.

**Comment by nitriques** [comment]
Are you missing a push ? I do not see your rebase... But thanks!!

**Comment by bernardodiasc** [comment]
Already sent, this one 5a831b4. Changes reflected in this PR.

**Comment by bernardodiasc** [comment]
fixed a typo, the correct commit is this 65ade63.

**Comment by nitriques** [comment]
All good !!
Should I wait for the fix for #15 ?

**Comment by bernardodiasc** [comment]
Oh, I don't even remember that issue. Let me check!

**Comment by nitriques** [comment]
All done! Thanks !!

**Pull request merged**